### ISSUE TITLE
Project logo?

### ISSUE THREAD

**Issue by oubiwann** [murarth/ketos#63]
I thought it might be fun to have a logo for the project ... I'll attach some ideas I had.

**Comment by oubiwann** [comment]
![Rust_Lisp1](https://user-images.githubusercontent.com/894251/58228994-09386280-7cf6-11e9-93a3-4df93ae70e59.jpg)

**Comment by oubiwann** [comment]
![Rust_Lisp2](https://user-images.githubusercontent.com/894251/58229008-11909d80-7cf6-11e9-8f51-a75ec2dcd248.jpg)

**Comment by oubiwann** [comment]
![Rust_Lisp3](https://user-images.githubusercontent.com/894251/58229013-17867e80-7cf6-11e9-9360-e7b8120e2bb4.jpg)

**Comment by oubiwann** [comment]
![Rust_Lisp4](https://user-images.githubusercontent.com/894251/58229062-3be25b00-7cf6-11e9-8cdf-f46ace5b118d.jpg)

**Comment by oubiwann** [comment]
![Rust_Lisp5](https://user-images.githubusercontent.com/894251/58229069-4270d280-7cf6-11e9-9ec8-370b3f512f73.jpg)

**Comment by oubiwann** [comment]
![Rust_Lisp6](https://user-images.githubusercontent.com/894251/58229075-48ff4a00-7cf6-11e9-95e2-1d8fa5998c39.jpg)

**Comment by oubiwann** [comment]
![Rust_Lisp7](https://user-images.githubusercontent.com/894251/58229086-51578500-7cf6-11e9-9b56-eea469ac389e.jpg)

**Comment by oubiwann** [comment]
![Rust_Lisp8](https://user-images.githubusercontent.com/894251/58229096-574d6600-7cf6-11e9-82af-97fdb20e45ba.jpg)

**Comment by murarth** [comment]
Wow. I love the sea monster one. That is amazing.

Is that original artwork? Did you make this?

**Comment by oubiwann** [comment]
Yeah, that’s my favorite one, too (my wife says “me too!”); the second sea monster has a lambda behind it ;-) I can’t take any credit for it, though — I looked for some free clip art that might capture the spirit of κῆτος, one that would make Poseidon smile ;-) (a google search for “sea dragon” did the trick). I vectorized it and modified it a bunch, so at this point, it’s derived “art” from free artwork.

I can submit a PR with the .svg file (what dir would you like it in?); I can also generate some .png files from it, if there are some sizes you’d like available in the repo ...

**Comment by oubiwann** [comment]
Btw, it gives me a giggle that our friend κῆτος more or less completely covering the lambda ... which, I think, makes it a bit of a secret lambda itself ;-)
